### PR TITLE
[Storage] [STG 97] Re-Record Existing Tests

### PR DESCRIPTION
**Pull request by weirongw23-msft**
These tests involve `create_file`, `set_http_headers`, and `upload_file`.

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py [hunk 1]
```diff
@@ -959,10 +959,6 @@ def _from_generated(cls, generated):
         props.change_time = generated.properties.change_time
         props.etag = generated.properties.etag
         props.permission_key = generated.permission_key
-        props.owner = generated.owner
```
Comment (weirongw23-msft) [inline sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py:962]: Removing `owner`, `group`, `file_mode`, `nfs_file_type`, and `link_count` from generated props as those don't exist in generated models.

**Review by jalauzon-msft — APPROVED** [review]

File: sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py (hunk 1 above)
Comment (jalauzon-msft) [reply sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py:962]: Do these ever actually get populated for directories? Like in `get_directory_properties`? If not, we should remove them entirely from this model.

Comment (weirongw23-msft) [reply sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py:962]: Yeah they get populated for `get_directory_properties`

Comment (jalauzon-msft) [reply sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py:962]: Ok, weird they aren't in the generated model then. This means they don't get populated for `list_files_and_directories`?

**Pull request merged**